src/components/render-settings-panel.js

```diff
@@ -25,6 +25,8 @@ import {Delete} from 'kepler.gl/components/common/icons';
 import ItemSelector from 'kepler.gl/components/common/item-selector/item-selector';
 import {Scene} from 'components/scene'; 
 
+import * as turf from '@turf/turf';
```

Comment:
I believe this is what you’d change switching to a sub package - look for the one that adds the “geometries”, like “point”.

Reply:
Addressed https://github.com/CodeLabs-Hubble-gl/kepler.gl/pull/36#discussion_r476700148